setup.py

```diff
@@ -22,7 +22,7 @@
 postgres_requires = sqlite_requires + ['psycopg2']
 mysql_requires = sqlite_requires + ['pymysql']
 
-omniscidb_requires = ['pymapd>=0.12.0']
+omniscidb_requires = ['pymapd==0.24']
```

Comment:
>= is probably resasonable here

Reply:
Afaik one equal only works for conda, if I'm not wrong double equal is required here.